src/page-support/exclusive-content/components/independent-script.js

```diff
@@ -0,0 +1,160 @@
+import React from 'react'
+import SyntaxHighlighter from 'react-syntax-highlighter'
+import { docco } from 'react-syntax-highlighter/dist/esm/styles/hljs'
+import { CopyContent } from './copy-content'
+
+const WORKER = 'https://webmonetization.org/api/exclusive-content'
+
+export function IndependentScript() {
+  const independentScript = `<script>
```

Comment:
Could this instead fetch the static script to avoid having to maintain two copies?